bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy

```diff
@@ -1387,10 +1419,17 @@ class ApiController {
         log.warn("Insert Document API called without a payload - ignoring")
         return;
       }
-      listOfPresentation << [name: "default", current: true];
+      if (hasPresentationUrlInParameter) {
+        if (!preUploadedPresentationOverrideDefault) {
+          listOfPresentation << [name: "default", current: false]
```

Comment:
```suggestion
          listOfPresentation << [name: "default", current: true]
```

In case `preUploadedPresentationOverrideDefault=false`, the `default.pdf` should be the current.